src/foam/u2/md/OverlayDropdown.js

```diff
@@ -0,0 +1,205 @@
+/** 
+ * @license
+ * Copyright 2017 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.u2.md',
+  name: 'OverlayDropdown',
+  extends: 'foam.u2.Element',
+
+  imports: [
+    'window'
+  ],
+
+  exports: [
+    'as dropdown'
+  ],
+
+  documentation: 'A popup overlay that grows from the top-right corner of ' +
+      'its container. Useful for e.g. "..." overflow menus in action bars. ' +
+      'Just $$DOC{ref:".add"} things to this container.',
+
+  properties: [
+    {
+      class: 'Float',
+      name: 'height',
+      value: 0
+    },
+    {
+      class: 'Boolean',
+      name: 'opened',
+      documentation: 'True when the overlay has been commanded to be open. ' +
+          'It might still be animating; see $$DOC{ref:".animationComplete"}.',
+    },
+    {
+      class: 'Boolean',
+      name: 'animationComplete',
+      documentation: 'True when an animation is running. The overlay hasn\'t ' +
+          'really reached the state commanded by $$DOC{ref:".opened"} until ' +
+          'this is true.',
+      value: true
+    },
+    {
+      name: 'dropdownE_',
```

Comment:
Do you need to build this in the factory (which means it might not get the proper context)? Can't you just build it like normal and then just specify dropdownE_$ as the third argument to start() or tag() to store the reference here?